docusaurus/docs/cms/configurations/environment.md

```diff
@@ -47,6 +47,15 @@ Strapi provides the following environment variables:
 | `TRANSFER_TOKEN_SALT` | Salt used for transfer tokens by the [Data Management](/cms/features/data-management) feature | `String` | `auto-generated` |
 | `DATABASE_CLIENT` | Database client to use (e.g., `sqlite`) | `String` | `sqlite` |
 | `DATABASE_FILENAME` | Location of the SQLite database file | `String` | `.tmp/data.db` |
+| `UP_JWT_MANAGEMENT` | [JWT management mode](/cms/features/users-permissions#jwt-management-modes) | `String` | 'legacy-support' |
+| `UP_SESSIONS_ACCESS_TTL` | JWT sessions management access token lifespan in seconds |  | `Number` | `604800` (1 week in seconds) |
+| `UP_SESSIONS_MAX_REFRESH_TTL` | JWT sessions management maximum refresh token lifespan in seconds| `Number` | `2592000`  (30 days in seconds) |
+| `UP_SESSIONS_IDLE_REFRESH_TTL` | JWT sessions management idle refresh token timeout in seconds | `Number` | `604800` (7 days in seconds) |
+| `UP_SESSIONS_HTTPONLY` | | `Boolean` |  `true` for HTTP-only cookies
```

Comment:
@jhoward1994 Could you please add description for the `UP_SESSIONS_HTTPONLY`, `UP_SESSIONS_COOKIE_NAME`, and `UP_SESSIONS_COOKIE_SECURE` entries of the table? And check "defaults" are correct (if defaults make sense in an env var context as such 🤷)